clusterloader2/testing/load/configmap.yaml

```diff
@@ -2,6 +2,9 @@ apiVersion: v1
 kind: ConfigMap
 metadata:
   name: {{.Name}}
+{{if not (eq (Mod .Index 20) 0 19) }} # .Index % 20 in {0,19} - only 10% deployments will have non-immutable ConfigMap.
```

Comment:
The if condition seems to be: `.Index % 20 not in {0,19}` which translates to 90% of deployments will have immutable Config map. 

It's basically what you wrote, but with less negations - easier to parse. 

NIT: Consider rewriting

Reply:
Let's merge as is for now and will clean up later - I'm slightly afraid of breaking something now and I would like that to land asap to have as much soaking time as possible.

Reply:
I proposed to change only the comment, but I'm fine with doing that later. Thanks!